pandas/tests/series/test_subclass.py

```diff
@@ -82,6 +84,7 @@ def test_subclass_sparse_addition(self):
         exp = tm.SubclassedSparseSeries([5., 7., 9.])
         tm.assert_sp_series_equal(s1 + s2, exp)
 
+    @pytest.mark.xfail(reason="XXX: SS used to reindex. Now we match Series.")
```

Comment:
This is a buggy test.

```python
In [8]: res
Out[8]:
   xxx
a    1
b    2
c
d

In [9]: exp
Out[9]:
   xxx
a    1
b    2
c
d

```

Now we (correctly) raise a ValueError about a shape mismatch, just like Series.

Reply:
Can this xfail be made strict?  (so we know to un-xfail it when it gets fixed)